CONTRIBUTING.md

```diff
@@ -129,6 +129,39 @@ If you plan to submit a pull request for anything more than a typo or obvious bu
 
 Depending on the size of the feature you may be expected to first write a design proposal. Follow the [Proposal Process](https://github.com/projectcontour/community/blob/master/GOVERNANCE.md#proposal-process) documented in Contour's Governance.
 
+### Issue and PR Triage
+
+#### Project Board
+
+In addition to maintaining the project repositories, project maintainers are responsible for maintaining the [project tracking board](https://github.com/orgs/projectcontour/projects/2).
+This board is intended to organize work for the team as well as provide visibility into the status and priority of tracks of work, specific Issues, and PRs.
+The board is used in conjunction with Issue and PR labels.
+
+Priority of cards on the board flows from left to right, with the *leftmost* column representing what is currently being worked on.
+Within a column, priority flows from top to bottom, with the *topmost* cards having the highest priority.
+
+The leftmost column should represent what is planned for the upcoming release.
+The "Investigating" column contains longer term items that may need more information, feedback from Issue reporters, etc.
+Further columns represent decreasing priority, with "Prioritized Backlog" containing cards that are coming soon, all the way to the "Unprioritized" column which contains items currently not yet sorted.
+
+**Notes for maintainers and contributors**
+- If you are looking for work to pick up, look to the leftmost columns of the project board
```

Comment:
Need a note somewhere that if it' assigned then someone is working on it. Folks should only assign work they are actively working on, otherwise need to unassign.